exercises/x86-64-assembly-1-b/README.md

```diff
@@ -1,6 +1,6 @@
 ## Introduction
 
-This is Part 1 of our research into how people write x86-64 assembly differently. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+This is Part 1 of our research into how people write x86-64 assembly differently. This exercise should take 10-15 minutes for a programmer who is proficient in x86-64 assembly. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
```

Comment:
Should "assembly" be captialized?

Reply:
They capitalize it on their track's page: https://github.com/exercism/x86-64-assembly/#exercism-x86-64-assembly-track